app/controllers/rentals_controller.rb

```diff
@@ -0,0 +1,39 @@
+class RentalsController < ApplicationController
+
+  def check_out
+
+    rental = Rental.new(rental_params)
+
+    if rental.movie.available_to_rent?
+      if rental.save
+        rental.movie.reduce_available_inventory
+        puts rental.customer.movies_checked_out_count
+        rental.customer.increase_movies_checked_out_count
+        render json: { id: rental.id, customer_id: rental.customer_id, movie_id: rental.movie_id }, status: :ok
+      else
+        render json: { errors: rental.errors.messages }, status: :bad_request
+      end
+    else
+      render json: { errors: rental.errors.messages }, status: :bad_request
+    end
+  end
+
+  def check_in
+
+    rental = Rental.find_by(rental_params)
+    if rental.save
+      rental.movie.increase_available_inventory
+      rental.customer.decrease_movies_checked_out_count
```

Comment:
All of this work could be wrapped up in one method, maybe something like `Rental.check_in`